docs/design-documents/hal/0005-pin-names-Arduino-Uno-standard.md

```diff
@@ -197,14 +210,14 @@ There should be both compile and run time checks to confirm whether a board has
 - PWM compatibility on `ARDUINO_UNO_D3/D5/D6/D9/D10/D11`
 - Analog compatibility on `ARDUINO_UNO_A0/A1/A2/A3/A4/A5`
 
-There is a proposal [here](https://github.com/ARMmbed/mbed-os/compare/master...jeromecoutant:DEV_STANDARDIZATION) on how to perform tests on pins.
 
-Additionally, there should be tests on each of the Arduino Uno pins to confirm whether the required funcionality is implemented correctly. This can be achieved by using the FPGA test shield and the existing [tests](https://github.com/ARMmbed/mbed-os/tree/master/hal/tests/TESTS/mbed_hal_fpga_ci_test_shield).
+A python script could check, during CI process, whether a board has valid ARDUIN_UNO pins defined (none equal to NC, and no duplicated pin values).
 
-The tests could be compiled and run unsing Greentea as shown here:
+Additionally, a Greentea test checks each of the Arduino Uno pins to confirm whether the required funcionality is implemented correctly (only API call check, no real check).
 
-    mbed test -t <toolchain> -m <target> -n *test_arduino_uno_pin_names* --compile
-    mbed test -t <toolchain> -m <target> -n *test_arduino_uno_pin_names* --run
+    mbed test -t <toolchain> -m <target> -v -n hal-tests-tests-pin_names-arduino_uno
+
+Real pin testing can be achieved by using the FPGA test shield and the existing [tests](https://github.com/ARMmbed/mbed-os/tree/master/hal/tests/TESTS/mbed_hal_fpga_ci_test_shield).
```

Comment:
"Real pin testing" --> I think we should rephrase to something like:
"Hardware signal testing on pins can be achieved ..."